scripts/modules/mozu-quotes-grid/mozuquotesgrid-view.js

```diff
@@ -31,6 +37,23 @@ define(["modules/jquery-mozu",
                     };
                 });
             },
+            userHasBehavior: function (behaviorId) {
```

Comment:
No need to write this comprehensive method, you can do shorthand them like below -

```
var behaviors = require.mozuData('user').behaviors;
this.model.set("isAdmin", behaviors.includes(1000) || behaviors.includes(1005));
```

Reply:
done

Reply:
If you're not using this method anymore then delete it. Don't say done and leave it here.